data/manager.php

```diff
@@ -41,6 +35,30 @@ public static function instance() {
 	 */
 	public $command_formats = [];
 
+	/**
+	 * Fix issue with 'Potentially polymorphic call. The code may be inoperable depending on the actual class instance passed as the argument.'.
+	 *
+	 * @return \Elementor\Core\Base\Module|\Elementor\Data\Manager
+	 */
+	public static function instance() {
+		return ( parent::instance() );
+	}
+
+	public function __construct() {
+		add_action( 'rest_api_init', function () {
+			// Empty server means: this is not an internal run.
+			if ( ! $this->is_internal && ! Error_Manager::is_registered() ) {
+				Error_Manager::register_handler( [ $this, 'rest_error_handler' ] );
+			}
+		} );
+	}
+
+	public function __destruct() {
+		if ( Error_Manager::is_registered() ) {
```

Comment:
Check `is_registered` from within the manager.

Reply:
Not relevant anymore.